memcached_stat.py

```diff
@@ -8,29 +9,24 @@
 # Global settings, one entry per instance
 CONFIGS = []
 
+class EmptyString:
```

Comment:
Yeah I should add a comment... basically if you put an empty string as password of the memcache client, it considers the parameter is missing and thus disable authentication (which we need for couchbase). So we need "something" that converts into an empty string and is not Null when testing it.